ArcadiaParties-SPA/src/app/app.component.scss

```diff
@@ -0,0 +1,11 @@
+.content-container {
```

Comment:
It should also have `flex` and `display` styles.

Reply:
I've fixed this problem.